ANSWERS.md

```diff
@@ -1,6 +1,12 @@
 ### Questions:
-1. What is the DOM?
+1. What is the DOM? 
+    Document Object Model
+
 2. What is an event?
+    Events are things the browser is listening for, to action something after a specific event occurs.
 3. What is an event listener?
+    its the method that is holding the Event
 4. Why would we convert a NodeList into an Array?
+    well a NodeList is a Array-like object.  So if we want to use anything in it we have to convert it into an array.
```

Comment:
This is both correct and incorrect. It is true that a NodeList is similar to an array but the reason we are converting one to the other is so we could use methods that are specific to arrays (that way we don't have to write them ourself!)